extensions/wikia/CommunityPage/scripts/CommunityPageBenefitsPageviews.js

```diff
@@ -14,9 +14,14 @@ require([
 		}
 	}
 
+	// Gets timestamp in format YYYY-mm-dd HH:mm:ss
+	function getTimestamp() {
+		return (new Date()).toISOString().substr(0,19).replace('T', ' ');
```

Comment:
Oops, I meant to put `yyyy-MM-dd HH:mm:ss` (what you said in slack) into comment. I think this output is identical to php function though